cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -334,7 +361,7 @@ private Cluster tryReuseCluster(DataprocClient client, ProvisionerContext contex
                 return false;
               }
               return true;
-            }).findAny();
+            }, new ErrorCategory(ErrorCategoryEnum.PROVISIONING)).findAny();
```

Comment:
nit: just use context.getErrorCategory. Less one object creation, consistent and shorter

Reply:
done.